docs/src/assets/landing.css

```
@@ -25,3 +25,63 @@
 [data-has-hero] .hero > img {
 	filter: drop-shadow(0 0 3rem var(--overlay-blurple));
 }
+
+/* Product Hunt launch banner — can be removed on 30 August 2023. */
+[data-has-hero] .sl-banner {
+	--sl-color-banner-bg: #f5574a;
```

Realised the PH orange is too light for good color contrast here. Fixing to meet WCAG requirements:
```suggestion
	--sl-color-banner-bg: #d73f37;
```